open-y-subprojects/openy_map
# fix: Filter locations by ID instead of title

From @szherebchuk, fixes #20.

To test:
- add a new location with the same title as an existing (Branch, Camp, or Facility), fill out the required fields and add some amenities
- visit /locations
- Test out amenities filters and observe filtering works properly.
